### DISCUSSION TITLE
Dataset issues for the DFN model

### DISCUSSION THREAD

**Discussion by MarcoZangrandi** [pybamm-team/PyBaMM#4037] [General]
Hello everyone, I'm new to this platform and Python in general. I'm using this library for my master's thesis. 
In this initial phase, I'm working on parameter sets, and my goal is to obtain 3 types of cells with different chemistries (NMC, NCA, LFP) but with the same cylindrical geometry (possibly 21700). These 3 parameter sets will then be used with the DFN model including a thermal model.

For the NMC type, I used the Chen2020 set, while for the LFP type, I was using the Prada set but I encountered some ambiguities that I can't explain. 

In particular, I can't understand where the electrode geometry comes from as it's not mentioned in the reference article. Compared to other sets, such as Chen2020, it's very different with the same geometry (60 cm vs. 6.5 cm in height and 0.3 m vs. 1.58 m in depth). Also, from the Prada article, here reported "A Simplified Electrochemical and Thermal Aging Model of LiFePO4-Graphite Li-ion Batteries: Power and Capacity Fade Simulations", it appears that the "Stoechiometry at 100% SOC" at the positive electrode is 0.035 compared to the 0.0038 used in the PyBaMM code (PyBaMM/pybamm/input/parameters/lithium_ion/Prada2013.py). 
I don't understand why these values were used, although the simulation using the PyBaMM set gives me a voltage curve in line with that type of chemistry.

Lastly, when I insert the thermal model, I need additional data such as collector thicknesses and other their characteristics like conductivity, density, etc. From your experience, since these values are not present in the article and in the set, would replacing them with the corresponding values from Chen2020 significantly compromise things? 

I've also been searching the internet for detailed datasets for DFN models but with little success. If you have any advice on this, I would appreciate it. 
Thank you very much for your patience and help.

**Comment by valentinsulzer** [comment]
> In particular, I can't understand where the electrode geometry comes from as it's not mentioned in the reference article. Compared to other sets, such as Chen2020, it's very different with the same geometry (60 cm vs. 6.5 cm in height and 0.3 m vs. 1.58 m in depth). 

The values in Prada2013 are just chosen to give the correct area of 0.18m2 from the paper. For DFN, only area matters, not height and width. Still, we should probably adjust them to be more realistic for a cylindrical cell form factor. Can you open a PR for that?

> Also, from the Prada article, here reported "A Simplified Electrochemical and Thermal Aging Model of LiFePO4-Graphite Li-ion Batteries: Power and Capacity Fade Simulations", it appears that the "Stoechiometry at 100% SOC" at the positive electrode is 0.035 compared to the 0.0038 used in the PyBaMM code (PyBaMM/pybamm/input/parameters/lithium_ion/Prada2013.py).
I don't understand why these values were used, although the simulation using the PyBaMM set gives me a voltage curve in line with that type of chemistry.

These were adjusted to give 3.6V at 100% SOC while maintaining the same total cyclable lithium.

> Lastly, when I insert the thermal model, I need additional data such as collector thicknesses and other their characteristics like conductivity, density, etc. From your experience, since these values are not present in the article and in the set, would replacing them with the corresponding values from Chen2020 significantly compromise things?

Using corresponding values is probably ok as a first approximation

> I've also been searching the internet for detailed datasets for DFN models but with little success. If you have any advice on this, I would appreciate it.

What kind of datasets?

**Comment by MarcoZangrandi** [comment]
>The values in Prada2013 are just chosen to give the correct area of 0.18m2 from the paper. For DFN, only area matters, not height and width. Still, we should probably adjust them to be more realistic for a cylindrical cell form factor. Can you open a PR for that?

Thank you very much for your answers. Not being an expert in electrochemical models I was not aware of this detail. As far as PR is concerned, as I have only recently started using python I wouldn't know how to help in this regard, sorry.

>What kind of datasets?

Sorry if I did not explain myself adequately. I have now tried to create, from literature papers, datasets based on the main parameters provided by Chen2020, so as to have 3 types of chemistry to supply to the DFN model, as the purpose will then be to build parallels and series with these cells. 
So the Chen2020 dataset works well, even with the thermal model, while the other types I found are missing data. 
So I was wondering if there are databases somewhere on the web that provide the essential parameters to be used in a DFN model (as for example we have in Chen2020) but for other chemistries and with a specific geometry (in this case cylindrical).
